tests/tests/cases.rs

```diff
@@ -9,7 +9,7 @@ use test_case::test_case;
 
 // Test cases for programs without input, it checks the outputs are correct automatically.
 
-// felt tests
+// // felt tests
```

Comment:
```suggestion
// felt tests
```

Reply:
Actually, scratch that:
```suggestion
```
That comment no longer describes what's going on below.